utils/merge_all_dependabot_PRs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This is a little tool for simplifying merging multiple dependabot RRs into one
+# as a workaround for the following feature request:
+# https://github.com/dependabot/dependabot-core/issues/1190
+# This is not meant to be fully automated and the final cherry-pick may require
+# user intervention to resolve conflicts.
+
+git fetch upstream
+for b in $(git branch -r --list upstream/dependabot/*) ;
+do
+  echo "Fetching last commit from $b"
+  c="$c $(git log HEAD~1..HEAD $b | awk '/^commit/{ print $2; exit;}')"
```

Comment:
use more descriptive name for `c`. e.g. `list_of_commits`, or add a comment at the top of the `for` loop summarising its function.

Reply:
Done.